images/v1alpha3_crs.drawio

```diff
@@ -0,0 +1 @@
KyuAdOxMFLX+KKWyZ49wEh70zxwjZoeM10FJPA+8TJP9iyzOkmgHGrL2qsmBIlamE+jo2ZM7Y4dwKmpkXUHyTwkVJCJDpCBcm4+KWKsnrOfFFZ74o4deYE3CLBnNOaGfrnDiB0Q6B80VNv7S55FyLaAsuuUBxsywV9MMlpX6Z7x7nWkAnvujnHJGNgTkhpcjECXztEDhfDHuOaO1MnJO0UcMlmtaPCxbvV5eH9Ktp8sQlfXJJDZqd95KBueTssi4Tl9DYS3AuqemXNpe0M4hOcskb0Vw0hhrtBe4ehoy5ziVN6sYJpZ7gaIr7BVtnmsZQpb1ExhAJcMp5PLUe+eRUx5T5RGeo8L4FPulvAyn1RJsTJqW235KDpmCaORGjG1uUusVve9LmikmJPc+xgq6LHjf34s1RHeoUcseDpMQYeAZw2owx6a0UGUPpkTFY7xgMVdWLZIyLccrrU9jOOZyglowWHRmh1JHEuIqxPsXp8MEIJTcaYz6YonR65YPSOu7MCMx3BJLdsv5icwJV6WJzdQlknLt839vuEFfVZKqx0Y19BN/Ce0FMVkwkQ05BU8xX42rAdNVTbNUiIRQm083U9GzPXy8mcB61y8V6baWeVgDKlw5guULoTCKkDWKKYboJ9IRhdbaAy8DQFKVS6idp6HVokKx23TFULxxD1TQEBSudRki1xhbCBia2OGdaJuzUkLwJUbNsalWJ05xgjW7UxJLWEt2vwUWwnhKq4TXLZpZSxkMyCIBQqwtjVCe4bJzdUCtmN2y3CrPkXuL1lw9oIHVpzzBu4SlPCrqxpOYvQWOXDi2pXChb0gwhjXjoO9OhbP/18eHf98pP95/14+31F/Hm62rWYAW+tUSHli4U75ZJolFemIoiCSYBqD7WJhGpLrln2yZrp5SOeCpM1Gbmx0LtFOac1HO8A1s/eF2DuFPqM03dgAPnUawaBiE5WL5EESEp4GztzurQYT1k354Dn1XhkxbjIjQRFl7SNKshPP9zQ9XCt0HgH2uHo2fDcgvDok/tiHdOulLS1pnWkbrIAmPNSUslk6qE4rXFJN1m8XGD1JsC+smNhFoe0nalaLoyRws2Z7j4ms+dwsRJm2H13v957Ps8oem5KK+iaf3x/eb+6+f7b/cPj/+9//rw4c/Pw0+OIyG/M6nnQRBBJREpn5ytrIYTYLMdOcFm4S0bVQYbE5mGLyjKTAEYhqI0jgdcoc+W8NDV1wZiWqpq/h/7ObDd9W28/r+B9da3I8CEmFxVwe0d2g4lF/mcyLSLjuD4xrdMvqpWsiAq1U0yKymTGEXxfbLzXsWzU0oqezdSl0Whth7B7aj04Qgm+jaqQ2rmKVVPGa7n4yC9pQhWzh9o4jgwrnxGRCKNzJkcgkfgNCwGTZJNworRdCG9SFaocmFSg6o6AGbUbrlxnHKDnkyTs25Mzjo+3S9RylPcjYzQxCcJmzV5a3s5Dio3a9qHfrbxJV+ziqKDJfYd/JUMWZBlQRTg+fQunvwOPghXTrCbBUG0tWeI7XaR8NMJfsmFsV4EfZwVcJ52+zVOJMswFXEuS8CVDdlWXFNTdUmbg7mmOos4n/VZlBiREqJZqMJ31RVQSSaEFiuyKWg6QbyiFm8lkULjMKUER7/BuT0OXUU3FCjNlu7wZwhpPKgqDe5hvzlVRS1etbBIS0mUBY0Quijjl2T7Q0qlHbvIQC3h3l9xmQ6FSR2Z1BHuBEE+Sdis6Z/A2xyW8T60o0ZV7fmcCGnpBS74CpYe2qcvbGonN5rf4DaDfmXR3QloZ3xEW6OASjHHb4IfN+7g728XTI0neEL5ILp/gdhfh8txREL32eT4wd6dbcPg2XNBErg5h8Ik4ge/JSk4Vk00XRPE4lUqlE2oJObKuipoBH2EosZIyvqCaYx1ctR8FBjoYrFcK6Z0EByK7LXDdplXkoKLhcwajUqq7SI7jLC8HXFbUltNvBpbpbW0m0zZ75qGieBrLvXVX7YNMhd0yrvSngveDKZYRzMD301pA9op7fPFA1rGoXv9S8aIpkEllBDNsiHxjqiCpb+TFDyvZeMcR3JdT7Qh7ZQh8/IgVSVNMLS+UCV0JtNLY0YGlmTIfYPAltZqCYjOa5U5pJ1SVb4BSEuZHLpvv6xP1E5JJ98ApBkyZ0Na6ok2pHT1mNFAqpZVyu6glvtivlLpajO5TKA5I0Vi6sgbKcZglkBwybgwJXXF3sL3aLwn2sh3qlJzgQvaUgTF6g1VpdiTSC8DPDlHU7tKIg7KLodyzXUAdlw2RQRzWjiuEpzGNijcmsUcZnnats84sRWrL+BLPdEGfjJsNASiO6SMJWulnUnj7WzZ9cg0xthUBJ0ZypvH5x8Pkvj31nlc+rsPyn8W7l12rS0PKiTzQ/I0CKNVsAw2tn9/bL05pp5FTrzjez4GwTYB8R8QRa8PcfLTO3sfBUVWAC9e9FfycfT4b/RY0JJndy+5l+5ec0++gNCDM0ZO/NR/WOa/Ijc1cUM2LBtA5EIipx4Y4BTZyQx8JmdaCrGE6WC8Vb+BjDAMuRQMQOCAMrLpB0TcYCKq5WS3WW7/QhRyH/cliTiRhLZCfE99bmO5mNu4GENDiPGAGPhffHtTEeFREZKDMU/PgR/nASuhtYdfXCak85eUY7rxvuM9iOi2So/bd8Q1hXQZ4mkuYh5djkXY05ixc5jqFk2VFGSfBTlC2e1Dlk8tjW0ckjiMg9FPJNk5EYyeUS8fj55Sb5YnPos5mCL6U54Dw9DqqiEQyBiCre859i45BzgbXTHxYIozIfcg07XSnEnP2U4qVmQxicA3tpcL2k2dATWoHlRDJivgjtIMjknO8mPwhgDdNGbjOEQJlH7KB54/cHnIEpqGuqx03qgJTXj+a05H7m28aKI3u5FvQ5Dpeeu1vXE7X1viYDLdmvoEipixj+HFpjONZoohpBGWmW1FI9jNVFkwCLkwutQFg0/DAGF9tIiG9nb1KXABesf/Aw==</diagram></mxfile>
```

Comment:
Is the image format correct ?

Reply:
There are three files:
 1 .png formatted
 2 .svg formatted
 3 .drawio formatted (this is needed so that everyone is able to modify it later on, without a need to create it from scratch)